.travis.yml

```diff
@@ -6,34 +6,55 @@ php:
   - 5.6
   - 7.0
 
+sudo: false
+
 env:
+  matrix:
+    - DB=mysql db_dsn='mysql://travis@0.0.0.0/cakephp_test'
+    - DB=pgsql db_dsn='postgres://postgres@127.0.0.1/cakephp_test'
+    - DB=sqlite db_dsn='sqlite:///:memory:'
   global:
     - DEFAULT=1
 
 matrix:
+  allow_failures:
+    - php: hhvm
+
   fast_finish: true
 
   include:
-  - php: 7.0
```

Comment:
Exactly the version does not matter there, it is just a lot faster. Unfortunately the php version usually does matter for the coverage.